src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.js

```diff
@@ -12,88 +12,87 @@ import Tooltip from '../Tooltip';
 import * as DeviceCapabilities from '../../libs/DeviceCapabilities';
 import styles from '../../styles/styles';
 import * as StyleUtils from '../../styles/StyleUtils';
-import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
-import {propTypes as anchorForCommentsOnlyPropTypes, defaultProps as anchorForCommentsOnlyDefaultProps} from './anchorForCommentsOnlyPropTypes';
+import {propTypes as anchorForCommentsOnlyPropTypes} from './anchorForCommentsOnlyPropTypes';
 import CONST from '../../CONST';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
 
 const propTypes = {
     /** Press in handler for the link */
+    // eslint-disable-next-line react/require-default-props
     onPressIn: PropTypes.func,
 
     /** Press out handler for the link */
+    // eslint-disable-next-line react/require-default-props
     onPressOut: PropTypes.func,
 
-    // eslint-disable-next-line react/forbid-prop-types
-    containerStyles: PropTypes.arrayOf(PropTypes.object),
-
     ...anchorForCommentsOnlyPropTypes,
-    ...windowDimensionsPropTypes,
-};
-
-const defaultProps = {
-    onPressIn: undefined,
-    onPressOut: undefined,
-    containerStyles: [],
-    ...anchorForCommentsOnlyDefaultProps,
 };
 
 /*
  * This is a default anchor component for regular links.
  */
-function BaseAnchorForCommentsOnly(props) {
+function BaseAnchorForCommentsOnly({onPressIn, onPressOut, href = '', rel = '', target = '', children = null, style = {}, onPress, ...rest}) {
```

Comment:
Can we use the defaultProps instead? Or is there a reason you took it out in favor of declaring them inline. The anchorForCommentsOnlyDefaultProps has these declared.